test/test_image.py

```diff
@@ -478,50 +479,50 @@ def test_write_jpeg_reference(img_path, tmpdir):
     assert_equal(torch_bytes, pil_bytes)
 
 
-# TODO: Remove the skip. See https://github.com/pytorch/vision/issues/5162.
-@pytest.mark.skip("this test fails because PIL uses libjpeg-turbo")
+@run_on_env_var(
```

Comment:
Only tangentially related to this PR, but do we really need this? Could we simply check the env var's value?

I spent a few minutes trying to understand what `get_bool_env_var` does, down to the `exists_ok` logic (which I still don't understand). I'm not sure we're winning in terms of code readability here?

Reply:
I've added docstrings. Let me know if they are not sufficient.